commands/create/cluster/command.go

```diff
@@ -239,6 +250,18 @@ func printResult(cmd *cobra.Command, positionalArgs []string) {
 		richError, richErrorOK := err.(*errgo.Err)
 
 		switch {
+		case IsHAMastersNotSupported(err):
+			headline = "Feature not supported"
+
+			requiredVersion := haMastersFeature.HAMasters.RequiredVersion(config.Config.Provider)
+			if requiredVersion != nil {
+				subtext = fmt.Sprintf("HA Masters is only supported by releases newer than %s.", *requiredVersion)
+			} else {
+				subtext = "HA Masters is not supported by your provider."
```

Comment:
```suggestion
				subtext = "Master node high availability is not supported on this provider."
```

Can we also add the provider name to the error message? I am not sure every user would know what the provider is in this context. Like `Master node high availability is not supported on this provider (KVM).`